css/style.css

```diff
@@ -472,16 +536,126 @@ textarea[type="text"]:hover {
   justify-content: center;
 }
 
-.social-icons {
-  display: flex;
-  margin-bottom: 20px;
+.form {
+  /* Center the form on the page */
+  margin: 0 auto;
+  width: 400px;
```

Comment:
Please notice that the form width is larger on the original design 😃:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199633361-4aadcd58-8286-4ced-a607-2973fbb3ee02.png)  |  ![image](https://user-images.githubusercontent.com/42799579/199633390-13b2f66c-c377-4e03-b4b0-5cbb6a90df73.png)  |

> Notice how your design has more blank space? 😉

Reply:
what about this code on the line 539 ?
do i need to change it ??

Reply:
> what about this code on the line 539 ? do i need to change it ??

Not exactly on line 539, but please update your .form to have the proper width. Notice that, in the original design, the form inputs are as wider as the text, and in your design they fit in the middle of the text 😉.